specs/FrameProcesssInfo.md

```diff
@@ -0,0 +1,503 @@
+Frame Process Info API
+===
+
+# Background
+Management of performance and overhead are some of the top concerns with 
+applications building more complex experiences using WebView2. We're seeing
+applications use multiple WebView2s, iframes, fencedframes, or other 
+embedding techniques for different aspects of their user's experience, and
+needing to prioritize or deprioritize the less important experiences depending
+on what the user is doing. This is similar to how the browser needs to 
+understand the impact of each tab in order to ensure that the background tabs 
+don't cause a major impact on the user's experience in the foreground tab.
+
+We provide the `GetProcessInfos` API for host applications to understand which
+processes are part of their WebView2s. That API provides enough information for
+a host application to understand the overall performance impact(memory, CPU 
+usage, etc.) of WebView2 on their application or the user's device, but it 
+doesn't provide the granularity needed for the host application to know which 
+part of WebView2 is consuming those resources.
+
+In this document we describe the updated API. We'd appreciate your feedback.
+
+# Description
+* We propose extending `CoreWebView2Environment` to include the 
+`GetProcessInfosWithDetails` API. This asynchronous call returns a 

```

Comment:
Confirming, getting these frame details needs to be async?  Is it expensive on the level of network or large-file access because of the multi-process model behind WV2?

Reply:
It can be expensive on the level of network when navigating across pages with child frames.

Reply:
In those cases, there could be a lot frame created/deleted message.